### DISCUSSION TITLE
Ability to block alternative browser downloads

### DISCUSSION THREAD

**Discussion by youofaallpeople** [proginosko/LeechBlockNG#547] [Q&A]
Hi,
I was wondering if anyone had any ideas about being able to block the downloading of alternative browsers using the same password that blocks the time restrictions for Leechblock.
Any advice appreciated!
Thanks,
Paul

**Comment by smayer97** [comment]
That is a VERY BROAD question. There are SO MANY ways to deal with alternate browsers. The approach depends on who you are trying to block and how rigorous a solution you are looking for. 

Short of using a third party software solution that controls everything that runs on a computer, the solution to resolving this is multi-pronged, and I might add, not perfect or bulletproof. 

One part of the solution is to make sure you have an admin account on the computer and prevent installation of ANY software without admin password and rights. The by-pass to this is that someone can use a portable browser (meaning, it does not require installation to run). So, the solution is to block access to USB ports to block the ability to run portable software (check out the free tool Simple Run Blocker for this). And to prevent downloading the software, one approach is to use a whitelist in LB and only allow access to permissible sites, thereby never being able to access a site with an alternative browser. Another piece is to try to limit which software can run on the computer. Both FREE tools I use include Simple Run Blocker and AskAdmin. Windows Firewall can also be used for some of the blocking, limiting which software can access the internet.
